packages/@okta/vuepress-site/docs/guides/telephony-inline-hook/main/index.md

```diff
@@ -35,7 +32,13 @@ This guide provides a working example of an Okta Telephony Inline Hook. It uses
 
 * Enable the Use Your Own Telephony Provider feature in your org. From the left navigation pane in the Admin Console, go to **Settings** > **Features**, locate the **Use Your Own Telephony Provider with the Telephony Inline Hook** slider, and slide to enable.
 
-> **Note:** If the feature doesn’t appear, contact [Okta Support](https://support.okta.com/help/s/) and make sure that SMS and Voice capabilities are enabled.
+    > **Note:** If the feature doesn’t appear, contact [Okta Support](https://support.okta.com/help/s/) and make sure that SMS and Voice capabilities are enabled.
+
+* Make sure that you have an active phone number in Twilio with SMS and MMS capabilities.
+
+* Create a [TwiML bin](https://www.twilio.com/docs/runtime/tutorials/twiml-bins#create-a-new-twiml-bin) in your Twilio account for use with Voice (Call) messages. You need the handler URL that is automatically generated to use as a variable.
+
+* Make sure that you have a user in your org that already has the [Phone authentication enrolled](https://help.okta.com/okta_help.htm?type=oie&id=ext-mfa-usage) with Okta.
```

Comment:
“Phone authentication enrolled” alias doesn’t appear to be working. I couldn’t find `ext-mfa-usage` in the alias sheet. Maybe it’s `ext-configure-phone`?

Reply:
it's brand new. It may not be live yet.